README.md

```diff
@@ -1,4 +1,4 @@
-# iotagent-json
+# FIWARE IoT Agent for a JSON-based Protocol
 
```

Comment:
Assuming that PR https://github.com/telefonicaid/iotagent-json/pull/351 is going to be merged before this one, a link to docker/README.md should be added.

Reply:
Fixed - 5c58c72